backend/api/prisonApi.ts

```diff
@@ -155,6 +155,8 @@ export const prisonApiFactory = (client) => {
 
   const getAgencies = (context) => get(context, '/api/agencies/prison')
```

Comment:
this is actually a get all prisons with contact details endpoint, something that isn't needed when we just want to get all prison descriptions